packages/fluent-theme/src/fluent/styles/CommandBar.styles.ts

```diff
@@ -0,0 +1,12 @@
+import { ICommandBarStyleProps } from 'office-ui-fabric-react/lib/CommandBar';
+
+export const CommandBarStyles = (props: ICommandBarStyleProps) => {
+  return {
+    root: [
+      {
+        height: 44,
```

Comment:
What was the previous height? If we can't avoid changing the components' dimensions (which sounds like the case here as you mention have discussed it with Ben) we should keep a list of these so that developers know where the theme may have introduced a breaking change to their layout.

Reply:
Aren't we planning to have the fluent theme first tested before making it default? This way when consumers will try out the fluent theme, they can see what broke in their layout and do the corresponding adjustments.
@betrue-final-final your comments would be much appreciated for clarification on what are the future plans regarding the design of `CommandBar`. I agree that changes that can influence external elements are dangerous but eventually if this is the intended design and we want to rollout first as an opt-in fluent theme consumers will not be taken by surprise.

Having a list is great but where we should keep it visible to anyone who decides to try out the fluent theme?

Reply:
Yes, we're doing this as a theme so that teams can opt-in to the changes and test before making it the default. I'm just suggesting that if we know of a change that is more likely to cause a broken layout we should make a note of it so that it's less likely to be missed while they test. Changing a gray slightly is unlikely to break anything, but component dimensions are often assumed to stay the same.

Reply:
Where that note would live? Perhaps on the fluent demo website?

Reply:
Let's put it in the wiki on Teams for now.

Reply:
@mikewheaton I totally get the issue. People can override the height, right? Ideally they wouldn't do that, and I'm not sure that commandbar will break from 4px difference, though. 

I do think that the new left nav has some relationship in the short term to the command bar still and Christopher's layout may get broken if we stick with 40. It would also be easier to deal with in a larger theme than as a separate bug fix, don't you think?

Reply:
@betrue-final-final I think we should change it to match the design, it's just good to make a note of it as (even with just a 4px difference) it's more likely to break a layout than something like a color or border change.